mesonbuild/coredata.py

```diff
@@ -1234,8 +1240,17 @@ def add_to_argparse(self, name: str, parser: argparse.ArgumentParser, help_suffi
     (OptionKey('auto_features'),   BuiltinOption(UserFeatureOption, "Override value of all 'auto' features", 'auto')),
     (OptionKey('backend'),         BuiltinOption(UserComboOption, 'Backend to use', 'ninja', choices=backendlist,
                                                  readonly=True)),
+    (OptionKey('genvslite'),
+     BuiltinOption(
+         UserComboOption,
+         'Setup multiple buildtype-suffixed ninja-backend build directories (e.g. builddir_[debug/release/etc.])'
+         'and generate [builddir]_vs containing a Visual Studio solution with multiple configurations that invoke a meson compile of the newly'
+         'setup build directories, as appropriate for the current build configuration (buildtype)',
+         'vs2022',
```

Comment:
The way Python string concatenation works is that these strings are joined _without_ a space between them, so instead of `newly setup` you get `newlysetup` instead.

Reply:
Thanks. Fixed in [1988526](https://github.com/mesonbuild/meson/pull/11049/commits/1988526a9196894e8f66ff86405c3870fb4d72a9)